src/Hl7.Fhir.Specification/Schema/Binding.cs

```diff
@@ -151,13 +152,13 @@ internal async Task<OperationOutcome> ValidateCode(ITypedElement source, Element
                 foreach (var issue in outcome.Issue) issue.Location = new string[] { location };
                 return outcome;
             }
-            catch (TerminologyServiceException tse)
+            catch (FhirOperationException tse)
             {
                 string message = (cc?.Coding == null || cc.Coding.Count == 1) 
                     ? $"Terminology service failed while validating code '{code ?? coding?.Code ?? cc?.Coding[0]?.Code}' (system '{system ?? coding?.System ?? cc?.Coding[0]?.System}'): {tse.Message}"
                     : $"Terminology service failed while validating the codes: {tse.Message}";            
 
-                return Issue.TERMINOLOGY_SERVICE_FAILED
+                return Issue.TERMINOLOGY_OUTPUT_WARNING
```

Comment:
The text in the error messages says "terminology service failed", but the `Issue` is now a `TERMINOLOGY_OUTPUT_WARNING`.  Since `TERMINOLOGY_SERVICE_FAILED` was already an issue of level `Warning` (which is what we want to achieve), I'd say we keep on using the `TERM_SERVICE_FAILED` issue - it did return a 4xx after all.